### PR TITLE
Updates to publications

### PR DESCRIPTION
**Pull request by loftusa**
Check for errors, I was unable to build and test myself.

### CODE REVIEW

File: content/pages/publications.html [hunk 1]
```diff
@@ -3,19 +3,24 @@
...
+- Browne2018
 - Wang2018
 - Wang2017
 - vogelstein2017linear
 - shen2016discovering
 - Tang2017
 - lee2017
 - Tang2016
-- tomita2015randomer
+- Tomita2018
```
Comment (falkben) [inline content/pages/publications.html]: This doesn't seem to reference anything

File: content/pubs/pubs.bib [hunk 2]
```diff
@@ -1,3 +1,29 @@
+@article{Browne2018,
+  author = {James Browne AND Tyler M. Tomita AND Disa Mhembere AND Randal Burns AND Joshua T. Vogelstein},
```
Comment (falkben) [inline content/pubs/pubs.bib]: the style is LASTNAME, FIRSTNAME MIDDLE-INITIAL.

File: content/pubs/pubs.bib [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+  title = Forest Packing: Fast, Parallel Decision Forests,
```
Comment (falkben) [inline content/pubs/pubs.bib]: Always need curly brackets

File: content/pubs/pubs.bib [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+  journal = {arXiv:1806.07300},
+  month = {June},
+  year  = {2018},
+  url = "https://arxiv.org/abs/1806.07300"
+}
+
+@article{Vogelstein2018equivalence,
+  author = {Cencheng Shen, Vogelstein, Joshua T.},
+  title = The Exact Equivalence of Distance and Kernel Methods for Hypothesis Testing,
```
Comment (falkben) [inline content/pubs/pubs.bib]: curly brackets

File: content/pubs/pubs.bib [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+  author = {James Browne AND Tyler M. Tomita AND Disa Mhembere AND Randal Burns AND Joshua T. Vogelstein},
+  title = Forest Packing: Fast, Parallel Decision Forests,
+  journal = {arXiv:1806.07300},
+  month = {June},
+  year  = {2018},
+  url = "https://arxiv.org/abs/1806.07300"
+}
+
+@article{Vogelstein2018equivalence,
+  author = {Cencheng Shen, Vogelstein, Joshua T.},
```
Comment (falkben) [inline content/pubs/pubs.bib]: author formatting order (lastname, firstname middle-initial)

**Comment by loftusa** [comment]
Should be good now

**Pull request merged**